awx/ui/src/locales/translations/fr/messages.po

```diff
@@ -0,0 +1,10713 @@
+msgid "Notification type"
+msgstr "Type de notification"
+
+#: components/NotificationList/NotificationList.js:177
+#: routeConfig.js:122
+#: screens/Inventory/Inventories.js:93
+#: screens/Inventory/InventorySource/InventorySource.js:99
+#: screens/ManagementJob/ManagementJob.js:116
+#: screens/ManagementJob/ManagementJobs.js:22
+#: screens/Organization/Organization.js:135
+#: screens/Organization/Organizations.js:33
+#: screens/Project/Project.js:114
+#: screens/Project/Projects.js:28
+#: screens/Template/Template.js:141
+#: screens/Template/Templates.js:46
+#: screens/Template/WorkflowJobTemplate.js:123
+msgid "Notifications"
+msgstr "Notifications"
+
+#: components/Schedule/ScheduleDetail/FrequencyDetails.js:169
+#: components/Schedule/shared/FrequencyDetailSubform.js:148
+msgid "November"
+msgstr "Novembre"
+
+#: components/StatusLabel/StatusLabel.js:44
+#: components/Workflow/WorkflowNodeHelp.js:117
+#: screens/Credential/shared/CredentialPlugins/CredentialPluginPrompt/CredentialPluginPrompt.js:66
+#: screens/Job/JobOutput/shared/HostStatusBar.js:35
+msgid "OK"
+msgstr "OK"
+
+#: components/Schedule/ScheduleOccurrences/ScheduleOccurrences.js:42
+#: components/Schedule/shared/FrequencyDetailSubform.js:549
+msgid "Occurrences"
+msgstr "Occurrences"
+
+#: components/Schedule/ScheduleDetail/FrequencyDetails.js:168
+#: components/Schedule/shared/FrequencyDetailSubform.js:143
+msgid "October"
+msgstr "Octobre"
+
+#: components/AdHocCommands/AdHocDetailsStep.js:189
+#: components/HostToggle/HostToggle.js:61
+#: components/LaunchPrompt/steps/OtherPromptsStep.js:192
+#: components/LaunchPrompt/steps/OtherPromptsStep.js:195
+#: components/PromptDetail/PromptDetail.js:362
+#: components/PromptDetail/PromptJobTemplateDetail.js:156
+#: components/Schedule/ScheduleDetail/ScheduleDetail.js:489
+#: components/Schedule/ScheduleToggle/ScheduleToggle.js:58
+#: screens/Inventory/SmartInventoryHostDetail/SmartInventoryHostDetail.js:45
+#: screens/Setting/shared/SettingDetail.js:98
+#: screens/Setting/shared/SharedFields.js:150
+#: screens/Template/JobTemplateDetail/JobTemplateDetail.js:284
+#: screens/Template/shared/JobTemplateForm.js:504
+msgid "Off"
+msgstr "Désactivé"
+
+#: components/AdHocCommands/AdHocDetailsStep.js:188
+#: components/HostToggle/HostToggle.js:60
+#: components/LaunchPrompt/steps/OtherPromptsStep.js:192
+#: components/LaunchPrompt/steps/OtherPromptsStep.js:194
+#: components/PromptDetail/PromptDetail.js:362
+#: components/PromptDetail/PromptJobTemplateDetail.js:156
+#: components/Schedule/ScheduleDetail/ScheduleDetail.js:489
+#: components/Schedule/ScheduleToggle/ScheduleToggle.js:57
+#: screens/Inventory/SmartInventoryHostDetail/SmartInventoryHostDetail.js:45
+#: screens/Setting/shared/SettingDetail.js:98
+#: screens/Setting/shared/SharedFields.js:149
+#: screens/Template/JobTemplateDetail/JobTemplateDetail.js:284
+#: screens/Template/shared/JobTemplateForm.js:504
+msgid "On"
+msgstr "Le"
```

Comment:
Based on [this PR](https://github.com/ansible/awx/pull/13048/files) the following change should occur:
```suggestion
msgstr "Activé"
```

Reply:
@AlexSCorey I have incorporated that change in this PR and requested that the translators add it to the translation memory.